python/invoke_release/tasks.py

```diff
@@ -1359,7 +1363,12 @@ def release(_, verbose=False, no_stash=False):
 
         if USE_PULL_REQUEST:
             _checkout_branch(verbose, current_branch_name)
-
+            try:
+                os.environ["GITHUB_TOKEN"]
+            except KeyError:
+                _standard_output("GITHUB_TOKEN env var not set. Unable to open a github PR")
+            else:
+                open_pull_request(branch_name, BRANCH_MASTER, release_version)
```

Comment:
This is not always true. When you are working against a MAJOR/MINOR branch, `BRANCH_MASTER` is incorrect here. I think the correct thing to use is `current_branch_name`.